Stratis.Bitcoin/Base/ChainState.cs

```diff
@@ -52,7 +53,10 @@ public bool IsInitialBlockDownload
                 if (this.lastupdate < this.fullNode.DateTimeProvider.GetUtcNow().Ticks)
                 {
                     this.lastupdate = this.fullNode.DateTimeProvider.GetUtcNow().AddMinutes(1).Ticks; // sample every minute
-                    this.lastresult = this.fullNode.IsInitialBlockDownload();
+
+                    // if consensus is no present IBD has no meaning. Set to false to match legacy code.
+                    var consensusLoop = this.fullNode.NodeService<ConsensusLoop>(true); // No hard failure if not present
```

Comment:
Will add IConsensusLoop and do something similar to below interfaces. Will use first matching. WDYT.